src/Makefile

```diff
@@ -785,6 +796,17 @@ $(SELINUX_POLICY): $(SELINUX_MODULE)
 $(SELINUX_MODULE): $(SELINUX_ENFORCEMENT)
 	checkmodule -M -m -o $@ $?
 
+ifeq (${TARGET}, agent)
+strip: ${BUILD_AGENT} ${BUILD_CMAKE_PROJECTS}
+else
+strip: ${BUILD_SERVER} ${BUILD_CMAKE_PROJECTS}
+endif
+ifeq (${uname_S}, Linux)
+ifeq (,$(filter ${DISABLE_STRIP_SYMBOLS},YES yes y Y 1))
+	@$(foreach var, ${EXECUTABLE_FILES}, objcopy --only-keep-debug $(var) symbols/$(basename $(notdir $(var))).debug; objcopy --strip-unneeded $(var);)
+endif
+endif
```

Comment:
A few things
- We need to mix TARGET and uname_S to determine the specific dependencies for strip target
- Ditto to the content of strip target